AmericanMedicalAssociation/AMA-style-guide
# EWL-3876 update modernizr

## Ticket(s)

NA

**Github Issue**

NA

**Jira Ticket**

- [EWL-3876: Style guide | flexbox class is manually declared, should be detected](https://issues.ama-assn.org/browse/EWL-3876)


## Description

- Synced modernizr with Drupal's version
- Removed fixed flexbox declaration in header
- Updated jQuery for kicks.

## To Test

- `gulp serve`
- Visit `http://localhost:3000/?p=templates-topic-grid-poc` and observe that the flex-based layouts still work and `flexbox` is still a class on the `html` element.
- Visit any item that uses jQuery [forms](http://localhost:3000/?p=organisms-simple-step-form) or [hub](http://localhost:3000/?p=templates-hub) and observe that all the jQuery things work and no console errors are thrown.


## Relevant Screenshots/GIFs

NA

## Remaining Tasks

None.

## Additional Notes

Anything more to add?
